python/pyspark/sql/connect/conversion.py

```diff
@@ -14,541 +14,31 @@
-
-            return convert_map
-
-        elif isinstance(dataType, BinaryType):
-
-            def convert_binary(value: Any) -> Any:
-                if value is None:
-                    return None
-                else:
-                    assert isinstance(value, bytes)
-                    return bytearray(value)
-
-            return convert_binary
-
-        elif isinstance(dataType, TimestampType):
-
-            def convert_timestample(value: Any) -> Any:
-                if value is None:
-                    return None
-                else:
-                    assert isinstance(value, datetime.datetime)
-                    return value.astimezone().replace(tzinfo=None)
-
-            return convert_timestample
-
-        elif isinstance(dataType, TimestampNTZType):
-
-            def convert_timestample_ntz(value: Any) -> Any:
-                if value is None:
-                    return None
-                else:
-                    assert isinstance(value, datetime.datetime)
-                    return value
-
-            return convert_timestample_ntz
-
-        elif isinstance(dataType, UserDefinedType):
-            udt: UserDefinedType = dataType
-
-            conv = ArrowTableToRowsConversion._create_converter(udt.sqlType())
-
-            def convert_udt(value: Any) -> Any:
-                if value is None:
-                    return None
-                else:
-                    return udt.deserialize(conv(value))
-
-            return convert_udt
-
-        elif isinstance(dataType, VariantType):
-
-            def convert_variant(value: Any) -> Any:
-                if value is None:
-                    return None
-                elif (
-                    isinstance(value, dict)
-                    and all(key in value for key in ["value", "metadata"])
-                    and all(isinstance(value[key], bytes) for key in ["value", "metadata"])
-                ):
-                    return VariantVal(value["value"], value["metadata"])
-                else:
-                    raise PySparkValueError(errorClass="MALFORMED_VARIANT")
-
-            return convert_variant
-
-        else:
-            return lambda value: value
-
-    @staticmethod
-    def convert(table: "pa.Table", schema: StructType) -> List[Row]:
-        assert isinstance(table, pa.Table)
-
-        assert schema is not None and isinstance(schema, StructType)
-
-        field_converters = [
-            ArrowTableToRowsConversion._create_converter(f.dataType) for f in schema.fields
-        ]
-
-        columnar_data = [column.to_pylist() for column in table.columns]
-
-        rows: List[Row] = []
-        for i in range(0, table.num_rows):
-            values = [field_converters[j](columnar_data[j][i]) for j in range(table.num_columns)]
-            rows.append(_create_row(fields=schema.fieldNames(), values=values))
-        return rows
+__all__ = [
```

Comment:
I think you can remove this. this conversion package isn't an API, and documented. It's just an internal package.

Reply:
If we remove this it will break forks that import from `pyspark.sql.connect.conversion` even if there's no merge conflict. Is that ok?

Reply:
`pyspark.sql.connect.conversion` isn't API so let's don't try to keep compatibility here.